bevy_lint/src/bin/main.rs

```diff
@@ -0,0 +1,38 @@
+use anyhow::{anyhow, ensure, Context};
+use std::{env, process::Command};
+
+fn main() -> anyhow::Result<()> {
+    // The `bevy_lint` lives in the same folder as `bevy_lint_driver`, so we can easily find it
+    // using the path of the current executable.
+    let mut driver_path = env::current_exe()
+        .context("Failed to retrieve the path to the current executable.")?
+        .parent()
+        .ok_or(anyhow!("Path to file must have a parent."))?
+        .join("bevy_lint_driver");
+
+    #[cfg(target_os = "windows")]
+    driver_path.set_extension("exe");
+
+    ensure!(
+        driver_path.exists(),
+        "Could not find `bevy_lint_driver` at {driver_path:?}, please ensure it is installed!",
```

Comment:
Use `driver_path.display()` to get a non-debug string. Also, could we send the user to a readme or such to help them?